cmd/sfncli/runner.go

```diff
@@ -65,12 +67,12 @@ func (t *TaskRunner) Process(ctx context.Context, args []string, input string) e
 		return t.sendTaskFailure(TaskFailureTaskInputNotJSON{input: input})
 	}
 
-	// convention: if the input contains _EXECUTION_NAME, pass it to the environment of the command
-	var executionName *string
-	if e, ok := taskInput["_EXECUTION_NAME"].(string); ok {
-		executionName = &e
-		t.logger.AddContext("execution_name", *executionName)
+	// convention: pass _EXECUTION_NAME into the environment of the command
```

Comment:
maybe add something about _EXECUTION_NAME being required in the task input

Reply:
called out in the README & code w/ cc475191afef07723d40bb56264811fa0cb8acee